src/VisualStudio/Core/Def/AnalyzerDependency/AnalyzerFileWatcherService.cs

```diff
@@ -38,37 +30,6 @@ internal sealed class AnalyzerFileWatcherService
 
     private readonly object _guard = new();
 
-    private readonly DiagnosticDescriptor _analyzerChangedRule = new(
-        id: IDEDiagnosticIds.AnalyzerChangedId,
-        title: ServicesVSResources.AnalyzerChangedOnDisk,
-        messageFormat: ServicesVSResources.The_analyzer_assembly_0_has_changed_Diagnostics_may_be_incorrect_until_Visual_Studio_is_restarted,
```

Comment:
❓ Do we still get this message displayed in the IDE?

Reply:
Currently (even without my PR) it will not.  That said, there are two paths (beyond this pr) to move forward on:

1. make it so that these diagnostics can proffered up so the user can see them.
2. change analyzers so that reloading is fine.  This is something we can support now, and @jasonmalinowski was going to make this work very soon anyways.